flow/peerdbenv/config.go

```diff
@@ -101,7 +101,7 @@ func PeerDBEncKeys() shared.PeerDBEncKeys {
 	return GetEnvJSON[shared.PeerDBEncKeys]("PEERDB_ENC_KEYS", nil)
 }
 
-func PeerDBCurrentEncKey() (*shared.PeerDBEncKey, error) {
+func PeerDBCurrentEncKey() (shared.PeerDBEncKey, error) {
```

Comment:
it's two strings, want to work with default encrypt/decrypt being nop, was having that work with `nil` but then key.ID is nil deref instead of empty string